react-native/gobuild.sh

```diff
@@ -59,10 +60,16 @@ rsync -pr --ignore-times "$go_client_dir/vendor" "$GOPATH/src/github.com/keybase
 rm -rf "$go_kbfs_dir/vendor"
 rm -rf "$go_client_dir/vendor"
 
+vendor_path="$GOPATH/src/github.com/keybase/vendor"
+gomobile_path="$vendor_path/golang.org/x/mobile/cmd/gomobile"
 
-if [ ! -f "$GOPATH/bin/gomobile" ]; then
-  echo "Build gomobile..."
-  go get golang.org/x/mobile/cmd/gomobile
+echo "Build gomobile..."
+(cd "$gomobile_path" && go build -o "$GOPATH/bin/gomobile")
+# The gomobile binary only looks for packages in the GOPATH,
+rsync -pr --ignore-times "$vendor_path/" "$GOPATH/src/"
```

Comment:
Is it possible to add another value to the `$GOPATH` env var instead of copying the files?